### PR TITLE
Fix shellcheck issues for scripts/e2e/e2e.sh

### PR DESCRIPTION
**Pull request by codenrhoden**
**What this PR does / why we need it**:
Fix shellcheck issues for scripts/e2e/e2e.sh

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: scripts/e2e/e2e.sh [hunk 1]
```diff
@@ -90,7 +90,7 @@ get_bootstrap_vm() {
 }
 
 export_base64_value() {
-   base64_value=$(echo -n "$2" | base64 -w 0)
+   base64_value=$(printf '%s' "${2}" | base64 -w0)
```
Comment (figo) [inline scripts/e2e/e2e.sh]: thanks for making the change.

is it possible to make `base64` works the same on MacOS and Linux?
currently `base64 -w` does not work on MacOS, i remember you did something similar in other change.

Comment (codenrhoden) [reply scripts/e2e/e2e.sh]: yeah I thought about doing that here too... I wasn't sure if this script was ever run on MacOS or not. but yeah, I can do that.

**Comment by codenrhoden** [comment]
@figo the `base64` change is now in place.

**Pull request merged**